### DISCUSSION TITLE
Slow response in initial load

### DISCUSSION THREAD

**Discussion by yurenklimbu** [vuejs/router#2073] [Help and Questions] [answered]
router-link slow response on initial load it took 7-8s but in only production. 
![slow](https://github.com/vuejs/router/assets/17329431/88a3375e-f8a3-400a-9417-70cc53004d2c)

**Comment by posva** [comment, marked as answer]
This could be many things but it's likely a blocking navigation guard with some API call. It's very unlikely to be at the router level.
